idaes/core/tests/test_control_volume_0d_scaling.py

```diff
@@ -147,7 +145,7 @@ def test_full_auto_scaling():
 
     # check that all variables have scaling factors
```

Comment:
It could be useful to put an inline comment here stating which variables do not have scaling factors.